packages/capabilities/src/index.js

```diff
@@ -0,0 +1,5 @@
+export * as Space from './capabilities/space.js'
+export * as Top from './capabilities/top.js'
+export * as Store from './capabilities/store.js'
+export * as Upload from './capabilities/upload.js'
+export * as Voucher from './capabilities/voucher.js'
```

Comment:
If you add black `types.js` you'd be able to do following to export all the types along with it.

```suggestion
export * as Voucher from './capabilities/voucher.js'

export * from './capabilities/type.js'
```

Reply:
If I add `types.js` next to `types.ts` it doesn't seem to work. we can try as followup but I think it might be trickier than that